commands/add-key.js

```diff
@@ -5,13 +5,13 @@ const { utils } = require('near-api-js');
 const checkCredentials = require('../utils/check-credentials');
 
 module.exports = {
-    command: 'add-key <account-id> <access-key>',
+    command: 'add-key <account-id>',
```

Comment:
I noticed there is another syntax to possibly use the existing arg as an optional parameter
```suggestion
    command: 'add-key <account-id> [access-key]',
```

example in the [generate-key command](https://github.com/near/near-cli/blob/4b4a787d0eafaa97fb7c2c741cdfaacf2c53f35b/commands/generate-key.js#L6)

This would allow to not break the current input of the command (and thus the existing docs). But I let maintainers decide for the best solution :)

Reply:
I think that's a pretty good idea. Other thoughs?